File: python/problem002.py
#Variables
fibonacciNumbers = [0,1]
answer = 0

#Finds fibanacci numbers
for i in range(0,4000000,2):

    #Finds value of next number
    x = fibonacciNumbers[-2] + fibonacciNumbers[-1]

    #Adds number to the list
    if x < 4000000:
        fibonacciNumbers.append(x)

        #Gets sum of even fibonacci number
        if x % 2 == 0:
            answer+=x

#Displays answer
print(answer) #Expected 4613732

File: javascript/problem002.js
//Variables
let fibonacciNumbers = [0, 1];
let answer = 0;

//Finds fibonacci numbers
for (i = 0; i < 4000000; i += 2) {

    //Finds value of next number
    let x = fibonacciNumbers[fibonacciNumbers.length - 2] + fibonacciNumbers[fibonacciNumbers.length - 1];

    //Adds number to the list
    if (x < 4000000) {
        fibonacciNumbers.push(x);

        //Gets sum of even fibonacci number
        if (x % 2 == 0) {
            answer += x;
        }
    }
}

//Displays answer
console.log(answer); //Expected 4613732